neurodiffeq/solvers.py

```diff
@@ -170,6 +195,9 @@ def make_pair_dict(train=None, valid=None):
             train=SamplerGenerator(train_generator),
             valid=SamplerGenerator(valid_generator),
         )
+        if self.generator['train']._requires_training:
+          self.generator_optimizer = Adam(chain.from_iterable([self.generator['train'].generator.net.parameters()]),lr=0.05)
```

Comment:
No need to use `chain.from_iterable`. Actually, I think the code should break here unless we remove the list square brackets. Am I missing something?